ansible/roles/grafana/meta/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+dependencies:
+
+  - role: ansible_plugins
```

Comment:
You should move this to the task list, use the `import_role` module.

Reply:
moved in tasks as import_role